types.d.ts

```diff
@@ -46,9 +46,11 @@ interface EmbedBlock extends Node {
 
 interface HTMLBlock extends Node {
   type: NodeTypes.htmlBlock;
+  children: Text[];
   data: Data & {
-    hName: 'html-block';
+    hName: 'HTMLBlock';
     hProperties: {
+      runScripts?: boolean | string;
```

Comment:
Per the above, this shouldn't really ever be a string!

```suggestion
      runScripts?: boolean;
```